### DISCUSSION TITLE
Can't get virtualbox running after update

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#427] [Q&A] [answered]
Hi there, I can't get virtualbox running. See screenshots.
EFI is enabled in my BIOS

Is there a way to get out of there?

Thanks

![Bildschirmfoto_2024-05-03_12-43-03](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/81ab235a-f86a-4ed8-b0aa-02a8fba45f2e)

![grafik](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/1b02f66f-0845-45de-bdad-d952969e9af0)

![grafik](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/eb5c4a42-f179-4b3a-93c4-b3ef3966e39f)

**Comment by geckolinux** [comment]
Hi there, is VirtualBox installed from the official Debian FastTrack repo?

**Reply by AlphaElwedritsch** [reply]
I did simply: `mkl@mkl-spirallinux:~$ sudo apt install virtualbox virtualbox-dkms virtualbox-qt`
Hope it is from official Debian FastTrack repo

**Reply by AlphaElwedritsch** [reply]
Yes, it's from fasttrack

![grafik](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/b8e3555b-5a18-4b22-aa2b-23688e747956)

**Comment by geckolinux** [comment]
It will probably work with the standard Debian Stable kernel if you could try booting with that.

**Reply by AlphaElwedritsch** [reply]
OK, how can I do that
The kernel I'm using was automatically installed.
I didn't explicitly select this one

**Reply by AlphaElwedritsch** [reply]
OK, found it in grub menu

But it's the same with 6.5.0-0.deb12.1-amd64


![grafik](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/0416848b-56fa-4656-9141-245f116f94a3)

**Comment by AlphaElwedritsch** [comment]
Cant get it installed on kernel 6.6 6.5 and 6.1
giving up now

**Reply by geckolinux** [reply]
Sorry I can't be of any help. I personally use VirtualBox without issues on my SpiralLinux installation.

**Reply by AlphaElwedritsch** [reply]
It's a fresh xfce installation. Without any modification.
Installed it two days ago, did a apt update, apt upgrade and tried to install virtualbox.
Nothing changed in repositories or synaptic.

That's why I'm so surprised.
that should work without any problem

Tomorrow I will give it a try with deb file from virtualbox download site

**Reply by geckolinux** [reply]
It would be good to report that to the FastTrack team:

- https://matrix.to/#/#debian-fasttrack:poddery.com
- https://salsa.debian.org/fasttrack-team/support/-/issues
- `team at fasttrack.debian.net`

**Reply by AlphaElwedritsch** [reply]
Tried to install the deb file from their Download page

Failed also

Maybe it is because I'm using UEFI secure boot on my laptop

And maybe that's why I can't install from fastrack

![grafik](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/16da93ef-1ce2-4b5c-9d6a-f648f4a4a940)

**Comment by AlphaElwedritsch** [comment, marked as answer]
Working now

It is an EFI Secure Boot issue

First tried it that way, but unsuccessfully...
https://wiki.debian.org/SecureBoot#MOK_-_Machine_Owner_Key

Switch then from Secure Boot to Legacy Mode in BIOS and could install VirtualBox
